web/crux/src/main.ts

```diff
@@ -20,6 +20,7 @@ import PrismaErrorInterceptor from './interceptors/prisma-error-interceptor'
 import prismaBootstrap from './services/prisma.bootstrap'
 import { PRODUCTION } from './shared/const'
 import DyoWsAdapter from './websockets/dyo.ws.adapter'
+import { coerce } from 'semver'
```

Comment:
Is this used?

Reply:
Yeah, i have forget that one, the linter thrown an error as well.